perception/autoware_tensorrt_bevdet/README.md

```diff
@@ -0,0 +1,69 @@
+# tensorrt_bevdet <!-- cspell: ignore bevdet -->
+
```

Comment:
Consider adding a brief, high-level description of the package's overall purpose in the very first line, under the title.  For example:  `tensorrt_bevdet: A ROS2 package for real-time 3D object detection using TensorRT and six surround-view cameras.` This provides immediate context for readers.